migrations/versions/12bb37c7d575_job_description_len.py

```diff
@@ -0,0 +1,38 @@
+"""job_description_len
+
+Revision ID: 12bb37c7d575
+Revises: 5e1c08ca7d4a
+Create Date: 2024-02-26 17:07:21.461612
+
+"""
+from alembic import op
+import sqlalchemy as sa
```

Comment:
5e1c08ca7d4a_job_location_id_none and 12bb37c7d575_job_description_len contains similiar logic of changing job.description length

Reply:
thanks for the review)